code/handlers/trackservers.q

```diff
@@ -235,6 +235,8 @@ addprocs:{[connectiontab;procs;connect]
 
 // addprocscustom is to allow bespoke extensions when adding processes
 addprocscustom:{[connectiontab;procs]}
```

Comment:
Just a quick explanation of why I changed it:

If I set
addprocscustom:@[value;`.servers.addprocscustom;{[connectiontab;procs]}]
It gives a rank error, for example if a variable called var is not defined the result of the error trap is a projection:
@[value;`var;{[x1;x2]}] 
q) {[x1;x2]}["var"]